positron/modules/atom_renderer_ipc.js

```diff
@@ -20,5 +20,15 @@ exports.sendSync = function(name, args) {
   // <http://electron.atom.io/docs/api/ipc-renderer/#ipcrenderersendsyncchannel-arg1-arg2->
   // expects to receive a single return value.  So this function returns
   // the first item in the array.
-  return cpmm.sendSyncMessage(name, args, { window })[0];
+  let result = cpmm.sendSyncMessage(name, args, { window })[0];
+  // If no one was listening for the message, wrap up the falsy result in the
+  // meta protocol used by Electron's RPC server.  This increases the chances we
+  // log an error specific to problem site, instead of here at this IPC code.
+  if (!result) {
+    result = JSON.stringify({
```

Comment:
is there some reason to not create a new variable here? It's a bit confusing how `value` is set to `result`, but then result gets trashed by the result of JSON.stringify. Will make things hard to debug in the future if need be.